packages/p2p/src/components/recommended-by/__tests__/recommended-by.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import RecommendedBy from '../recommended-by';
+
+jest.mock('Components/modal-manager/modal-manager-context', () => ({
+    ...jest.requireActual('Components/modal-manager/modal-manager-context'),
+    useModalManagerContext: () => ({
+        showModal: jest.fn(),
+    }),
+}));
+
+describe('<RecommendedBy />', () => {
+    it('it should show `No one has recommended this trader yet` and `recommended_average` equals to 0% if there is no props passed', () => {
+        render(<RecommendedBy />);
+        userEvent.click(screen.getByTestId('dt_popover_wrapper'));
```

Comment:
can we use fireEvent instead? if it works then we can avoid adding the new package.

Reply:
The tests are failed with fireEvent. userEvent library has been used in other packages as well